app/policies/department_policy.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class DepartmentPolicy < ApplicationPolicy
+
+  attr_reader :user
+  attr_reader :department
+
+  def initialize(user, department)
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+    @user = user
+    @department = department
+  end
+
+  def admin_new?
```

Comment:
update these to match the new action names in the controller (e.g. new, update, etc.)